src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -1067,13 +1071,15 @@ void HydrofractureSolver< POROMECHANICS_SOLVER >::initializeNewFractureFields( D
             {
               initialPressure = std::min( initialPressure, fluidPressure_n[fractureElementIndex] );
```

Comment:
I wonder why we use the `fluidPressure_n`of the neighboring fracture element instead of `fluidPressure`? Same question for the `fluidDensity`.

Reply:
after a propagation stage we should reset all those variables and resolve so I would expect these two to be the same.